### DISCUSSION TITLE
4GB Leech Split Size not working

### DISCUSSION THREAD

**Discussion by uploadtele** [anasty17/mirror-leech-telegram-bot#923] [Q&A]
hey i tried to leech 8GB file using leech option but the bot is splitting 2GB instead of 4GB, i tired on different accounts which has premium in it. any solution on how to fix it? i updated the git repo as well no solution yet.

**Comment by anasty17** [comment]
Deploy again.  Maybe pyrogram version is old

**Reply by uploadtele** [reply]
yes tried still same issue, its splitting it only 2GB 
![image](https://user-images.githubusercontent.com/106587107/185093253-06b72e19-60a4-4bb9-9547-b713bf0c602b.png)

**Comment by anasty17** [comment]
Don't crop the the screenshot let me see the sender

**Reply by uploadtele** [reply]
fixed bro i didn't read readme properly, the solution is i have to add user session to the config.env

**Reply by TheFlashSpeedster** [reply]
> fixed bro i didn't read readme properly, the solution is i have to add user session to the config.env

Very good 😂
You Didn't read readme and expect to work?
